net-misc/bopm/bopm-3.1.3-r2.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils user
+
+DESCRIPTION="Blitzed Open Proxy Monitor"
+HOMEPAGE="http://www.blitzed.org/bopm/"
+SRC_URI="http://static.blitzed.org/www.blitzed.org/${PN}/files/${P}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~ppc ~x86"
+IUSE=""
+
+RDEPEND=""
+DEPEND="${RDEPEND}"
+
+PATCHES=("${FILESDIR}"/remove-njabl.patch)
+
+src_configure() {
+	econf \
+		--datadir="${EPREFIX}"/usr/share/doc/${PF} \
```

Comment:
bopm actually does something weird here, so instead I documented it with a comment